### PR TITLE
Implement dark mode config for auth ui v2 customization

### PR DESCRIPTION
**Pull request by elise-ng**
Ref DEV-1560 (part 1/2)

Implemented todos except "only save diff from defaults"

https://github.com/user-attachments/assets/7f632ea4-1466-40eb-abfe-b759340514cd

### CODE REVIEW

**Comment by elise-ng** [comment]
@louischan-oursky

**Review by louischan-oursky — COMMENTED** [review]
@pkong-ds A heads up. This PR may conflict with some of your PR.

**Comment by elise-ng** [comment]
@louischan-oursky all updated and fixed the secondary button issue mentioned on slack by removing the unused entries

File: pkg/auth/handler/webapp/viewmodels/base.go [hunk 1]
```diff
@@ -245,7 +246,8 @@ func (m *BaseViewModeler) ViewModel(r *http.Request, rw http.ResponseWriter) Bas
 			url, _ = m.StaticAssets.GeneratedStaticAssetURL(id)
 			return
 		},
-		DarkThemeEnabled: !m.AuthUI.DarkThemeDisabled,
+		DarkThemeEnabled:  !m.AuthUI.DarkThemeDisabled || webapp.IsInlinePreviewPageRequest(r),
+		LightThemeEnabled: !m.AuthUI.LightThemeDisabled,
```
Comment (louischan-oursky) [inline pkg/auth/handler/webapp/viewmodels/base.go]: It should be `!m.AuthUI.LightThemeDisabled || webapp.IsInlinePreviewPageRequest(r)`

**Review by louischan-oursky — APPROVED** [review]

**Comment by fungc-io** [comment]
See this link for "Auto" behaviour: https://www.figma.com/design/IgOg9OzuMAmyIizN1CVeAt/Q2-%2F-2024?node-id=2016-19216&t=wLmUlICFU2UPk7fM-4
- both light & dark options are shown
- user can choose to preview the light or dark by a toggle button

**Comment by fungc-io** [comment]
Change radio button wordings to

- Light mode only
- Dark mode only
- Auto based on user preference

**Pull request merged**